book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
+
+.. code-block:: bash
+
+    $ composer require 'sensio/distribution-bundle'
```

Comment:
Do you know if there is a preview tool i can use for this documentation? I am checking on http://rst.ninjs.org/ at the moment and it displays ok but not as precise as I d like